engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/util/TypeToDisplayTextNode.java

```diff
@@ -38,11 +38,11 @@ String doDisplay(
       // Most likely it should be removed once that is implemented.
       return "null";
     } else if (TypesGen.isLong(value)) {
-      return value + " (Integer)";
+      return value.toString();
     } else if (TypesGen.isEnsoBigInteger(value)) {
-      return "Integer";
+      return value.toString();
     } else if (TypesGen.isDouble(value)) {
-      return value + " (Decimal)";
```

Comment:
Add specializations to `AnyToDisplayTextNode` and reverted the changes to `TypeToDisplayTextNode`.
I have amended the `to_display_text` of `No_Such_Method` to get the type object instead of the value.